From 73c44645147db2730976c9003b20ace61c3a5f3d Mon Sep 17 00:00:00 2001
From: GuaiYiHu <guaiyihu@foxmail.com>
Date: Fri, 16 Oct 2026 06:34:40 +0800
Subject: [PATCH] libc++: Allow the scoped MiOne legacy vendor interpreter

The device interpreter builds current Bionic with an exact legacy-daemon allowlist. It needs the same noexcept archive as the platform linker because the interpreter cannot use ELF TLS for exception globals. Limit access to the device interpreter package.

Change-Id: I70efb82b164641603315ab9f49925c380517a6de
---
 Android.bp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Android.bp b/Android.bp
index c5e337c..3241cae 100644
--- a/Android.bp
+++ b/Android.bp
@@ -421,6 +421,7 @@ cc_defaults {
     visibility: [
         "//bionic:__subpackages__",
         "//frameworks/libs/native_bridge_support:__subpackages__",
+        "//device/xiaomi/mione_plus/compat/linker",
     ],
 }
 
